user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Template_Editor.md

```diff
@@ -0,0 +1,338 @@
+
+To access the Template Editor:
+
+- For the grid and timeline components:
+
+  1. Select the component and go to the *Layout* tab on the right.
+
+  1. In the *Item templates* section, click *Edit*.
+
+     > [!NOTE]
+     > A preview of the template currently used for this component is displayed in the *Item templates* section.
+
+- For the table component:
+
+  1. Select the component and go to the *Layout* tab on the right.
+
+  1. In the *Column appearance* section, click the downward arrow next to *Select a column* and select the column you want to create a template for.
+
+  1. Click the ellipsis button ("...") in the top-right corner and select *Customize preset*.
+
+The Template Editor UI consists of the following main components<!--RN 34858-->:
+
+- Side pane:
+
+  - The *Layers* tab (1A)
+
+  - The *Tools* tab (1B)
+
+- Preview (2)
+
+- *Settings* pane (3)
+
+![Template Editor](~/user-guide/images/Template_Editor.png)<br/>*The Template Editor in DataMiner 10.4.1*
+
+> [!IMPORTANT]
+> Any changes made in the Template Editor are only saved once you click the *Save* button in the lower right corner. When you close the Template Editor by clicking the "X" in the top-right corner or the *Cancel* button in the lower right corner, you will be alerted of any unsaved changes that may be lost if you close the editor prematurely.
+
+## Working with layers
+
+In the side pane of the Template Editor, the *Layers* tab offers an overview of all layers. Layers are used in the Template Editor to make non-destructive edits by stacking icons, text, rectangles, and ellipses one on top of the other without compromising the pixels of individual elements.
+
+The benefits of working with layers:
+
+- You can add elements on top of each other without removing data.
+
+- You can apply conditions to only one layer of the template. See [Adding conditional cases to a layer](#adding-conditional-cases-to-a-layer).
+
+- You can discard any unsatisfactory layers without affecting the entire template.
+
+- You can mask layers so that they are temporarily hidden without the need to delete them.
+
+- ...
+
+If one layer is positioned higher than another, the higher layer will appear in front of the lower one. To **change the order of layers**, click the ![*Drag-and-drop*](~/user-guide/images/drag-and-drop.png) button and drag the layer up or down in the *Layers* tab. Release the mouse button when you reach the position where you want to place the selected layer.
+
+![Change order layers](~/user-guide/images/Change_Order_Layers.gif)<br/>*The Template Editor in DataMiner 10.4.1*
+
+To **mask a layer**, hover the mouse pointer over the layer and click the *eye* icon in the lower right corner. The layer is now no longer visible in the template's end result. Clicking the *eye* icon once more makes the layer visible again. Alternatively, you can mask the layer by adjusting its properties in the *Settings* pane. See [Specifying layer properties](#specifying-layer-properties).
```

Comment:
The button on the layers in the left tab only hides the layer in the template editor. When saving, the visibilty of the layer only depends on the setting in the right sidebar. 

The button on hover in the layer is purely for 'designing' purposes

Reply:
Feedback implemented in commit [c5a531e](https://github.com/SkylineCommunications/dataminer-docs/pull/2328/commits/c5a531ea38eb524b55d42150f16caa60890a4dc5).